code/game/objects/items/weapons/tools.dm

```diff
@@ -580,6 +605,19 @@
 /obj/item/weapon/weldingtool/mini/flamethrower_screwdriver()
 	return
 
+/obj/item/weapon/weldingtool/abductor
+	name = "alien welding tool"
+	desc = "An alien welding tool. Whatever fuel it uses, it never runs out."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "welder"
+	toolspeed = 0.1

```

Comment:
honestly the ce roundstart tools should be slower, 4X speed is really good on its own; these should probably be like 0.2 or 0.25